src/lib/style/base/_core.scss

```diff
@@ -1205,7 +1205,7 @@ hr ~ .SRC-wrapper {
   padding: 0px;
 }
 
-#addAndRemoveColumns ~ div.dropdown-menu {
+#addAndRemoveColumns-dropdown.dropdown-menu {
```

Comment:
The style was previously applied relative to the tooltip element. When the Tooltip implementation changed, this style broke.